doc/command-line.md

```diff
@@ -0,0 +1,138 @@
+
+# Using sourmash from the command line
+
+
+From the command line, sourmash can be used to compute [MinHash
+sketches][0] from DNA
+sequences, compare them to each other, and plot the results.  This
+allows you to estimate sequence similarity quickly and accurately.
+
+Please see the [mash software][1] http://mash.readthedocs.io/en/latest/__
+ and the [mash paper (Ondov et al., 2016)][2]
+for background
+information on how and why MinHash sketches work.
+
+______
+
+sourmash uses a subcommand syntax, so all commands start with
+`sourmash` followed by a subcommand specifying the action to be
+taken.
+
+Contents
+
+* [An example](## An example)
+* [The `sourmash` command and its subcommands](## The sourmash command and its subcommands)
+  * [`sourmash compute`](### `sourmash compute`)
+
+  * [`sourmash compare`](### `sourmash compare`)
+
+  * [`sourmash plot`](### `sourmash plot`)
```

Comment:
It looks like this block isn't formatted correctly.

Reply:
I think you probably want the `.. contents::` directive here. You've manually pasted in some links that should be dynamically generated by the documentation build system

Reply:
I *may* have been mistaken here too. If the `.. contents::` directive doesn't work in Markdown, we'll have to figure out how to add anchors to the different sections—or simply just delete the contents block!